### ISSUE TITLE
Problem building with Piranha

### ISSUE THREAD

**Issue by jppelteret** [symengine/symengine#1083]
I'm putting together a package for the [Spack package manager](https://github.com/LLNL/spack), and have hit a problem when building with the Piranha CAS library. I'm not quite sure whether this is a symengine problem or a piranha problem. I've attached the [build log](https://github.com/symengine/symengine/files/504445/spack-build.txt) for your perusal, with the salient lines being
```
/private/var/folders/lc/b4bjrtcn69d279m6rf4f42y00000gn/T/jp/spack-stage/spack-stage-3YZfHb/symengine/symengine/polys/uintpoly_piranha.h:217:19: error: expected unqualified-id
        static Cf PZERO(0);
                  ^
/usr/include/sys/param.h:124:15: note: expanded from macro 'PZERO'
#define PZERO   22              /* No longer magic, shouldn't be here.  XXX */
```

**Comment by jppelteret** [comment]
Oh, and I forgot to note that this happens for both the development version and archived version `v0.2.0`.

**Comment by certik** [comment]
Thanks for reporting the problem @jppelteret. I think this should be fixed by #1084. I'll leave this open until that PR is merged.
